File: FLLandBaseViewController(OC)/FLLandBaseViewController(OC)/SecondViewController.m
//
//  SecondViewController.m
//  FLLandBaseViewController(OC)
//
//  Created by clarence on 16/9/9.
//  Copyright © 2016年 clarence. All rights reserved.
//

#import "SecondViewController.h"

@interface SecondViewController ()

@end

@implementation SecondViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    NSLog(@"我里面什么都不用处理了~~~");
    
    UILabel *label = [[UILabel alloc] initWithFrame:CGRectMake(self.view.bounds.size.width - 300, 100, 300, 30)];
    label.text = @"小咧咧 clarence";
    [self.view addSubview:label];
}

@end

File: FLLandBaseViewController(Swift)/FLLandBaseViewController(Swift)/SecondViewController.swift
//
//  SecondViewController.swift
//  FLLandBaseViewController(Swift)
//
//  Created by clarence on 16/9/9.
//  Copyright © 2016年 clarence. All rights reserved.
//

import UIKit

class SecondViewController: FLLandBaseViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let label = UILabel.init(frame: CGRect.init(x: self.view.bounds.size.width - 200, y: 0, width: 200, height: 30))
        label.text = "小咧咧 clarence"
        view.addSubview(label)
    }

}
